### PR TITLE
Remove imp and just use importlib to avoid memory error when importin…

### LINKED ISSUE #13282: pandas.show_versions causing malloc_error_break
If blosc is installed show_versions() will caused python to produce a malloc_error_break (double free) error on exiting. This can be seen in the travis runs under `source activate pandas && ci/print_versions.py` section but can also be easily replicated on Linux and Mac. The fix for this is to replace imp (which has been deprecated) with importlib. I will create and link a PR.

```python
>>> import pandas
>>> pandas.show_versions()

INSTALLED VERSIONS
------------------
commit: e0a2e3bc51f9e178f72c44e6de06700ee0bf31c6
python: 2.7.11.final.0
python-bits: 64
OS: Darwin
OS-release: 15.5.0
machine: x86_64
processor: i386
byteorder: little
LC_ALL: None
LANG: en_GB.UTF-8

pandas: 0.18.1+66.ge0a2e3b
nose: 1.3.7
pip: 8.1.1
setuptools: 20.7.0
Cython: 0.24
numpy: 1.11.0
scipy: 0.17.0
statsmodels: None
xarray: None
IPython: 4.1.2
sphinx: None
patsy: None
dateutil: 2.5.2
pytz: 2016.3
blosc: 1.3.2
bottleneck: None
tables: None
numexpr: None
matplotlib: 1.5.1
openpyxl: None
xlrd: 0.9.4
xlwt: None
xlsxwriter: None
lxml: None
bs4: None
html5lib: None
httplib2: None
apiclient: None
sqlalchemy: None
pymysql: None
psycopg2: None
jinja2: 2.8
boto: None
pandas_datareader: None```

>>> 
python(1726,0x7fff77074000) malloc: *** error for object 0x102573a00: pointer being freed was not allocated
*** set a breakpoint in malloc_error_break to debug
Abort trap: 6

### PR DESCRIPTION
**Pull request by nparley**
- [ ] closes #13282

Remove imp from pandas.show_versions() to fix memory problem.

### CODE REVIEW

**Comment by jreback** [comment]
hmm, ``importlib`` is only available in PY3 I think.

can you hack something from this:
https://pypi.python.org/pypi/importlib (don't want to add as a dep)

**Comment by nparley** [comment]
import lib is in 2.7 but this is for python2.6?

**Comment by jreback** [comment]
oh, then that's ok

File: pandas/util/print_versions.py [hunk 1]
```diff
@@ -99,11 +98,8 @@ def show_versions(as_json=False):
     deps_blob = list()
     for (modname, ver_f) in deps:
         try:
-            try:
-                mod = imp.load_module(modname, *imp.find_module(modname))
-            except (ImportError):
-                import importlib
-                mod = importlib.import_module(modname)
+            import importlib
```
Comment (jreback) [inline pandas/util/print_versions.py]: was wondering does this be here or can be a top-level import?

Comment (nparley) [inline pandas/util/print_versions.py]: No it should be a top-level import now you are right.

**Comment by jreback** [comment]
ok this looks good. ping when green.

**Comment by jreback** [comment]
thanks @nparley

**Pull request closed without merging**